### PR TITLE
Update moveit config to match setup assistant

### PR DESCRIPTION
**Pull request by nbellowe**

### CODE REVIEW

File: clam_moveit_config/launch/clam_moveit_controller_manager.launch [hunk 1]
```diff
@@ -1,21 +0,0 @@
...
-  <param name="controller_manager_name" value="$(arg controller_manager_name)" />
-
-  <!-- Flag indicating whether the controller manager should be used or not -->
-  <arg name="use_controller_manager" default="true" />
-  <param name="use_controller_manager" value="$(arg use_controller_manager)" />
-
-  <!-- load controller_list and controller_manager_ns -->
-  <rosparam file="$(find clam_moveit_config)/config/clam_execution_controllers.yaml"/>
-  
-</launch>
```
Comment (davetcoleman) [inline clam_moveit_config/launch/clam_moveit_controller_manager.launch:21]: This file should not be overwritten - it is important for connecting to the robot

**Comment by davetcoleman** [comment]
Please restore that file, then this can be merged.

**Comment by nbellowe** [comment]
OK, will do!

**Comment by nbellowe** [comment]
Done, whenever you're ready.

**Comment by davetcoleman** [comment]
Awesome sauce

**Comment by davetcoleman** [comment]
No, that should not have been deleted @nbellowe 

Restored: https://github.com/davetcoleman/clam/commit/fb883a02c0bd5d9c0d7aac00facd7c4cbe303eaa

Thanks for the heads up!

**Pull request merged**